### ISSUE TITLE
Confirm "Français" spelling

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#442]
On the following screens, please ensure "Français" is spelt correctly with the cédille “Ç”.

1. When the user launches the app for the first time, and have the option to select language.
2. On the settings > Language, where the user has the option to change the language.

**Comment by weglmar** [comment]
@ForumG-SC No, it should remain as Français.
My understanding of this ticket was that the accessibility label is not using the ç and instead says Francais.

**Comment by weglmar** [comment]
@Rajat-Sen I'm unable to validate this one (not sure if I'm doing it wrong). Could you please confirm? Thank you :)

**Comment by Rajat-Sen** [comment]
Not saying it correctly for some reason, will take some more time to test this one properly.

**Comment by weglmar** [comment]
@ForumG-SC it sounds like it is reading "francaise" and not "Français," could you please confirm it is Français? 
Rajat had changed his operating settings to French and it still does not sound correct!

**Comment by weglmar** [comment]
@ForumG-SC 

Also, on this screen, it is reading it as "select French - button" but should say Français

![image](https://user-images.githubusercontent.com/98898969/223160154-4b1417cb-a0a8-4226-b90a-7277cb18b9f8.png)

**Comment by weglmar** [comment]
@ForumG-SC When the app is in English it should read "Select Français button" when in French, it should read "Sélectionnez Français bouton"

**Comment by weglmar** [comment]
Validated on iPhone
